scale/docs/rest/scan.rst

```diff
@@ -0,0 +1,535 @@
+|        ]                                                                                                                |
+|    }                                                                                                                    |
++-------------------------------------------------------------------------------------------------------------------------+
+
+.. _rest_scan_create:
+
++-------------------------------------------------------------------------------------------------------------------------+
+| **Create Scan**                                                                                                         |
++=========================================================================================================================+
+| Creates a new Scan process and places it onto the queue                                                                 |
++-------------------------------------------------------------------------------------------------------------------------+
+| **POST** /scans/                                                                                                        |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **Content Type**   | *application/json*                                                                                 |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **JSON Fields**                                                                                                         |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| name               | String            | Required | The identifying name of the Scan process used for queries.          |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| title              | String            | Optional | The human readable display name of the Scan process.                |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| description        | String            | Optional | A longer description of the Scan process.                           |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| configuration      | JSON Object       | Required | JSON defining the Scan configuration.                               |
+|                    |                   |          | (See :ref:`architecture_scan_spec`)                                 |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
```

Comment:
Nope, as that will be identified in the `ingest` parameter given to the `scan/{id}/process` API call.